terraform/main.tf

```diff
@@ -160,13 +121,13 @@ resource "google_cloud_scheduler_job" "megalista_job" {
   name             = "megalista_job"
   description      = "Daily Runner for Megalista"
   schedule         = "0 0 * * *"
-  time_zone        = "America/Sao_Paulo"
+  time_zone        = local.time_zone
   attempt_deadline = "320s"
-  region           = "us-central1"
+  region           = var.region
 
   http_target {
     http_method = "POST"
-    uri         = "https://dataflow.googleapis.com/v1b3/projects/${data.google_client_config.current.project}/templates:launch?gcsPath=gs://${var.bucket_name}/templates/megalist"
+    uri         = "https://dataflow.googleapis.com/v1b3/projects/${data.google_client_config.current.project}/templates:launch?gcsPath=gs://${var.bucket_name}/templates/megalista"
```

Comment:
Template name actually needs to be megalist. See https://github.com/google/megalista/blob/f51d296e11ccee9cebf76d3940cb03513b73e19d/run_cloud.sh#L24

Reply:
Oops. just saw that you also modifed run_cloud.sh script. I guess it makes sense to keep consistency here. Nevermind this comment.